### PR TITLE
Meny customer admin

### PR DESCRIPTION
**Pull request by tantting**
## Description

Provide a clear description of what the PR does and why its relevant

## Related Issue ( optional )

Link to any related issue

## Changes

Describe the changes in more detail, if necessary include explanation of why some choices were made.

## Screenshots & GIFs ( optional )

media to provide more visual context if necessary

## Notes & Comments

General notes and comments, such as if there is things to consider ahead of deployment

### CODE REVIEW

File: RebelAllianceBank/Classes/Bank.cs [hunk 1]
```diff
@@ -1,10 +1,216 @@
-﻿namespace RebelAllianceBank.Classes
+﻿using System.Diagnostics;
+using System.Xml.Schema;
```
Comment (JosefForkman) [inline RebelAllianceBank/Classes/Bank.cs:2]: Remove unused code

**Review by JosefForkman — CHANGES_REQUESTED** [review]
Move all menu logic to a own class. Otherwise a good code

**Comment by tantting** [comment]
REquested changes have been done and pushed.

**Review by JosefForkman — APPROVED** [review]
Much better when the code is in its own class

**Review by JosefForkman — APPROVED** [review]

**Review by JosefForkman — APPROVED** [review]

**Review by Darkdusk234 — APPROVED** [review]
Well Done

**Pull request merged**